### DISCUSSION TITLE
Save and load the graph data

### DISCUSSION THREAD

**Discussion by songsong0425** [pyg-team/pytorch_geometric#7627] [Q&A] [answered]
Hi, always thank you for your effort on the PyG.
I have a simple question about the loading graph data.

During practicing the graph-based deep learning model, I found it cumbersome to create PyG graph data from scratch (i.e., from dataframe to PyG Data).
So I want to save the graph Data as a file and load it like .csv or .pkl and saw the method for it. (https://discuss.pytorch.org/t/pytorch-geometric-can-i-save-a-data-object-to-a-file/78546)
But I'm not sure this is the best way to save and load the data. Are there any other methods for this work?

**Comment by rusty1s** [comment, marked as answer]
Yes, `torch.save` and `torch.load` is the recommended way to store `Data` objects in PyG.

**Reply by songsong0425** [reply]
Thank you for your rapid reply. I'll try to use them! :)
